### PR TITLE
Create weekly-update.2015-04-17.md

### PR DESCRIPTION
**Pull request by Gioyik**
CC #55

### CODE REVIEW

**Comment by c0b41** [comment]
@Gioyik When will the merge?

**Comment by Gioyik** [comment]
@ayhankuru Let's wait for more people review it, then someone with the rights will merge it. :)

**Comment by rosskukulinski** [comment]
Overall looks good -- thanks for pulling this together @Gioyik.  I'm wondering how we should reference the pending 1.8 release ([ongoing here](https://github.com/iojs/io.js/issues/1436) including draft release notes).  We are including the 1.8 perf improvements, but if its not out yet, then I think we should bundle that comment with the actual 1.8 release notes.

I'm thinking that we shouldn't wait for 1.8 to land for us to release this weekly update.  In that case, lets hold onto the 1.8 crypto performance comment until next week.

Thoughts?

**Comment by Gioyik** [comment]
I think is OK in the way we present it. It's on Comunity Updates section and not presented as something released. But we can keep it for a week update when we mention 1.8 features. I don't have problem with it.

**Comment by diagramatics** [comment]
I wonder, should we use H3 for Notable Changes and Known Issues or H4? I found the rendered Markdown, when pasted to Medium automatically formats it right like the rest of the published articles with H4.

And speaking of Known Issues, none for this week?

File: weekly-updates/weekly-update.2015-04-17.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+* io.js mention on [Oracle's blog](https://blogs.oracle.com/java-platform-group/entry/node_js_and_io_js).
+* State of the io.js Build [April 2015](https://github.com/iojs/build/issues/77)
+
+### Upcoming Events
+
+* [JSConf Uruguay](http://jsconf.uy) tickets are on sale, April 24th & 25th at Montevideo, Uruguay
+* [NodeConf Adventure](http://nodeconf.com/) tickets are on sale, June 11th - 14th at Walker Creek Ranch, CA
+* [CascadiaJS](http://2015.cascadiajs.com/) tickets are on sale, July 8th - 10th at Washington State
+* [NodeConf EU](http://nodeconf.eu/) tickets are on sale, September 6th - 9th at Waterford, Ireland
+* [nodeSchool Tokyo](http://nodejs.connpass.com/event/13182/) will be held in April 12th at Tokyo, Japan
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-04-17.md]: NodeSchool Tokyo is finished.

**Comment by Gioyik** [comment]
> And speaking of Known Issues, none for this week?

Not this time: https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md#2015-04-14-version-171-rvagg

**Comment by diagramatics** [comment]
Hm, weird. 1.8.0 has the same issues as 1.6.4. Is it intentional?

**Comment by Gioyik** [comment]
@diagramatics not sure about it, but seems something to be discussed for next weekly update due this week is about 1.7.x

**Comment by yosuke-furukawa** [comment]
I guess Known Issues is just forgotten ... 
We have to include the Known issues in this article. I will fix the change log and throw a pull request to io.js.

If we remove them, everyone misleads the known issues are resolved.

**Comment by Gioyik** [comment]
@yosuke-furukawa which know issues we should add? Same as in previous weekly update?

**Comment by yosuke-furukawa** [comment]
@Gioyik 
Here. https://github.com/iojs/io.js/pull/1473
The known issues are seems to be good. But does not merge yet.
io.js core collaborators is discussing on how to handle the Known issues.

@Gioyik go ahead to write down the Known issues and publish the medium ??? this is Wednesday!!!

**Comment by Gioyik** [comment]
@yosuke-furukawa know issues are added. Let me know if there's something to add.

**Comment by yosuke-furukawa** [comment]
LGTM.

**Comment by Gioyik** [comment]
I have not permissions to merge PR (I am not added to org) and publish on Medium. So from now it's a task of someone else :)

**Comment by yosuke-furukawa** [comment]
I don't have a permission to publish on Medium neither. @julianduque @mikeal please publish our weekly updates on medium :)

**Comment by Gioyik** [comment]
Show us your super powers \o/

**Comment by julianduque** [comment]
Published: https://medium.com/node-js-javascript/io-js-week-of-april-17th-e4c6f2db7659

Sorry for the late publication

**Comment by diagramatics** [comment]
@julianduque No subtitles for this week on Medium?

(I feel bad for nitpicking all over the place)

**Comment by julianduque** [comment]
@diagramatics you are the official nit picker ;) and done, added subtitles

**Comment by diagramatics** [comment]
Great, thanks! :)

**Pull request merged**